sources/web/datalab/static/datalab.css

```diff
@@ -215,14 +215,14 @@ div.btn-group a {
 div.btn-toolbar {
   margin: 0px;
 }
-div.btn-toolbar .toolbar-btn {
+div.btn-toolbar .toolbar-btn, #help_menu_container .dropdown-toggle {
   background-color: transparent;
   border: none;
   font-family: 'Open Sans', Helvetica, sans-serif;
   font-weight: normal;
   font-size: 14px;
   outline: solid 1px transparent !important;
-  height: 46px;
+  /* height: 46px; */
```

Comment:
Was this just temporarily commented out for testing?

We should not have commented out lines in the final commit.